Sources/Bow/Transformers/OptionT.swift

```diff
@@ -95,6 +95,22 @@ extension OptionT where F: Functor {
     public func subflatMap<B>(_ f: @escaping (A) -> Option<B>) -> OptionT<F, B> {
         return transform { option in Option.fix(option.flatMap(f)) }
     }
+
+    /// Convert this `OptionT` to an `EitherT`.
+    ///
+    /// - Parameter defaultRigth: Function returning a default value to use as right if the `OptionT` is none.
+    /// - Returns: Returns: An `EitherT` containing the value as left or as right with the default value if the `OptionT` contains a none.
+    public func toLeft<R>(_ defaultRigth: @escaping () -> R) -> EitherT<F, A, R> {
+        return EitherT(cata({ .right(defaultRigth()) }, Either.left))
```

Comment:
There is a spelling mistake in `Right`.
```suggestion
        return EitherT(cata({ .right(defaultRight()) }, Either.left))
```

Reply:
You can also replace `{ .right(defaultRight()) }` by `constant(.right(defaultRight()))`, but no big deal, as you prefer.

Reply:
Using `@autoclosure` seems a good idea if we are fine with the syntax change for the caller, because we can still use `function()` as a parameter and it will not be executed, however (correct me if I'm wrong) using `constant` in the implementation makes me feel that we are losing the advantage of using a function, because we are going to execute the `.right(defaultRight())` and `defaultRight` could be a time-consuming function or could contain a side effect.

Let me know what you think. In fact, I would change `constant` to be implemented by using @autoclosure, in this way we can get the lambda without executing the `constant`.

Reply:
Take into account that we are not using `@autoclosure` in the library right now, we have a lot of cases that are similar to this one and we are not using it. I'm not against the decision of using it, I just want to verify we want it.